src/main/java/org/jboss/set/draw/Lottery.java

```diff
@@ -106,7 +106,7 @@ public static String createEmailText(String email, List<Issue> issues) {
                                 projectLinksEntry.getValue().stream()
                                         .map(issue -> LINK_LINE.formatted(issue.getBrowseUri().toString()))
                                         .collect(Collectors.joining()))
-                        .collect(joining("\n")),
+                        .collect(joining("<br>")),
```

Comment:
```suggestion
                        .collect(joining("<br>\n")),
```

So the string represantation is still human-readable